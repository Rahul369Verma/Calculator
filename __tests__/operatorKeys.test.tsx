import React from "react";
const renderer = require("react-test-renderer");
import OperatorKeys from "../components/operatorkeys";

it("renders NumericKeys correctly", () => {
  const tree = renderer.create(<OperatorKeys handleClick={jest.fn()} />);
  expect(tree).toMatchSnapshot();
});
