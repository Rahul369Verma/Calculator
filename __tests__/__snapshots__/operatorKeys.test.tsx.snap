// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NumericKeys correctly 1`] = `
[
  <button
    className="btn-cal btn-top"
    onClick={[Function]}
    type="button"
  >
    AC
  </button>,
  <button
    className="btn-cal btn-top"
    onClick={[Function]}
    type="button"
  >
    CE
  </button>,
  <button
    className="btn-cal btn-top"
    onClick={[Function]}
    type="button"
  >
    +/-
  </button>,
  <button
    className="btn-cal btn-top"
    onClick={[Function]}
    type="button"
  >
    %
  </button>,
  <button
    className="btn-cal btn-side"
    onClick={[Function]}
    type="button"
  >
    รท
  </button>,
  <button
    className="btn-cal btn-side"
    onClick={[Function]}
    type="button"
  >
    x
  </button>,
  <button
    className="btn-cal btn-side"
    onClick={[Function]}
    type="button"
  >
    -
  </button>,
  <button
    className="btn-cal btn-side"
    onClick={[Function]}
    type="button"
  >
    +
  </button>,
  <button
    className="btn-cal btn-side"
    onClick={[Function]}
    type="button"
  >
    =
  </button>,
]
`;
